frontend/app/assets/javascripts/spree/frontend/checkout/payment.js

```diff
@@ -1,5 +1,11 @@
 //= require spree/frontend/coupon_manager
 Spree.ready(function ($) {
+  function enableSubmitButton() {
```

Comment:
@kaliszKrzysztof you can use this existing method - https://github.com/spree/spree/blob/98751433c01709d90ffc32f41c822fcd1b277413/frontend/app/assets/javascripts/spree/frontend/checkout.js#L14